scripts/run-ui.sh

```diff
@@ -4,12 +4,15 @@ export FM_BITCOIN_NETWORK="regtest"
 export FM_UI_KIND=${1:-"old"}
 export FM_FED_SIZE=${2:-2}
 export FM_FED_NAME=${3:-"Cypherpunk Federation"}
+export RUST_BACKTRACE="full"
```

Comment:
why full? RUST_BACKTRACE=1 has more readable backtrace.

Reply:
Whoops, left this in from debugging a panic I ran into while running this. I'll remove.

Reply:
```suggestion
```
Just commenting so I get alerts when it's merged.